.github/workflows/codespell-private.yml

```diff
@@ -7,6 +7,10 @@ jobs:
   make-check-dictionaries:
     runs-on: ubuntu-latest
     steps:
+      - name: Set up Python
+        uses: actions/setup-python@v2
+      - name: Install dependencies
+        run: pip install pytest pytest-cov pytest-dependency
```

Comment:
Should we make a requirements-test.txt or something so we can load one file for GitHub and Travis?

Reply:
Feel free to add it if you want

Reply:
This seems to be one way to go:
https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies

So I've done it